### PR TITLE
Separate packaging and translations better.

### PR DESCRIPTION
**Pull request by m3nu**
I made some changes to the contributor docs to document the new macOS packaging workflow.

Also noticed that some Flatpak instructions are probably duplicate and moved some translation stuff from the main page to the translation page.

@samuel-w , can you check if both Flatpak sections in *Packaging* are needed? Here: https://github.com/borgbase/vorta.borgbase.com/blob/b49fd8c06b122b639901d914517c81103a7334d8/contributing/packaging.md

How are the two Flatpak sections related?

### CODE REVIEW

File: contributing/packaging.md [hunk 1]
```diff
@@ -7,8 +7,27 @@ parent: Contributing
...
+After downloading the app bundle, unzip it and open it via right-click > open to get around Gatekeeper.
+
+Once you are happy with the app, sign and package it as DMG locally using `make dist/Vorta.dmg`
+
+
 ## Linux with Flatpak
 
+The recommended way to build Vorta on Linux is via Flatpak. This can be done by following these [instructions](https://wiki.gnome.org/Newcomers/BuildProject).
+
+They first guide you through the installation of the IDE GNOME Builder. GNOME Builder is then used to clone, build and run Vorta.
```
Comment (samuel-w) [inline contributing/packaging.md:27]: This is a duplicate of the instructions below.

Comment (samuel-w) [reply contributing/packaging.md:27]: Line 50
`$ flatpak-builder --repo=$HOME/my-flatpak-builds app com.borgbase.Vorta.yaml`

File: contributing/translations.md [hunk 2]
```diff
@@ -53,6 +60,23 @@ Translations are managed on [Transifex](https://www.Transifex.com/borgbase/vorta
...
+Translated:
+- GUI texts / messages
+
+Add new strings for translation:
+- In Qt (sub)classes, use self.tr("English string"), scope will
+  be the instance class name.
+- Elsewhere use vorta.i18n.translate("scopename", "English string")
+- To only mark for string extraction, but not immediately translate, use 
+  vorta.i18n.trans_late function. Later, to translate, use vorta.i18n.translate (giving same scope).
+
```
Comment (samuel-w) [inline contributing/translations.md:78]: Add note here about string has to be marked with `trans_late` for `translate` to work later

Comment (samuel-w) [reply contributing/translations.md:78]: Sidenote: Why do we have separate `trans_late` and `translate`?

Comment (m3nu) [reply contributing/translations.md:78]: Not sure why Thomas did it like that. One is a dummy to mark the string. Maybe we can make it just one in the future.

I'll revisit this later and see if this needs changing or merge the functions.

```
def translate(*args, **kwargs):
    """
    small wrapper around QCoreApplication.translate()
    """
    global application  # see init_translation
    return application.translate(*args, **kwargs)


def trans_late(scope, text):
    """
    dummy translate function - only purpose is to mark scope/text for message extraction.

    later, at another place, there will be a translate(scope, text) call, with same
    scope and text, potentially not literal scope, not literal text, but from a variable.
    """
    return text
```

Comment (m3nu) [reply contributing/translations.md:78]: Ah, I think he needed this to translate e.g. settings.

**Pull request merged**